BenchmarkVCFs/SimpleBenchmark.wdl

```diff
@@ -0,0 +1,596 @@
+struct Reference {
+    File fasta
+    File index
+    File dict
+}
+
+# Object representing expression for subsetting VCFs
+struct VariantSelector {
+    String bcf_label
+    String bcf_selector
+}
+
+# Object representing file with intervals to subset analysis over
+struct Stratifier {
+    String label
+    File? interval
+}
+
+# Main workflow: performs evaluation of call_vcf against base_vcf using vcfeval comparison engine,
+# over different analysis modes.
+workflow SimpleBenchmark {
+    input {
+        # VCF information
+        File base_vcf
+        File base_vcf_index
+        String base_output_sample_name
+        String? base_vcf_sample_name
+
+        File call_vcf
+        File call_vcf_index
+        String call_output_sample_name
+        String? call_vcf_sample_name
+
+        # Reference information
+        File ref_fasta
+        File ref_index
+        File ref_dict
+
+        # Subsetting inputs using intervals
+        Array[File] strat_intervals = []
+        Array[String] strat_labels = []
+        Int interval_padding = 0
+        String subset_gatk_tag = "4.2.3.0"
+
+        # Subsetting inputs using variant properties; Note SNP & INDEL are separated automatically later
+        Array[String] bcf_selectors = []
+        Array[String] bcf_labels = []
+
+        # Evaluation inputs
+        File? evaluation_bed
+        Array[String] score_fields = ["GQ"]
+
+        # Columns to add to output files
+        String? experiment_label
+        String? extra_column
+        String? extra_column_label
+
+        # Null File type -- do NOT assign; Needed until WDL has better support for null File values
+        File? NULL_FILE
+    }
+
+    # Create reference object
+    Reference reference = {"fasta": ref_fasta, "index": ref_index, "dict": ref_dict}
+
+    # Add in default "Whole Genome" empty interval list
+    Array[File] full_strat_intervals = flatten([[""], strat_intervals])
+    Array[String] full_strat_labels = flatten([[""], strat_labels])
+
+    # Make Stratifier objects
+    scatter (strat in zip(full_strat_labels, full_strat_intervals)) {
+        Stratifier stratifier_list = {"label": strat.left, "interval": if strat.right != "" then strat.right else NULL_FILE}
+    }
+
+    # Make VariantSelector objects, with default None, Het, and HomVar types
+    Array[String] default_bcf_selectors =  ["", "GT='het'", "GT='hom'"]
+    Array[String] default_bcf_labels = ["", "Het", "HomVar"]
+    Array[String] full_bcf_selectors = flatten([default_bcf_selectors, bcf_selectors])
+    Array[String] full_bcf_labels = flatten([default_bcf_labels, bcf_labels])
+    scatter (selection in zip(full_bcf_labels, full_bcf_selectors)) {
+        VariantSelector selector_list = {"bcf_label": selection.left, "bcf_selector": selection.right}
+    }
+
+    # Perform analysis over all Stratifiers
+    scatter (stratifier in stratifier_list) {
+        call SubsetVCF as SubsetEval {
```

Comment:
This is definitely one option, and then have the `evaluation_bed` into for `vcfeval` to be over those intersections. I think the only tradeoff in that case is longer unnecessary compute time, especially when the stratifiers are small, since `vcfeval` reads the entire VCF it's given each time. But since there will always be one shard reading over the entire thing for the default empty stratifier, this probably won't add _too_ much extra time to the workflow. (A small amount might be added from increased quota usage? Maybe?) 

One of the reasons I liked using SelectVariants as well was that it allows for a flexible range of interval list types. Trying to eliminate that step means normalizing everything to bed files. There's also the `--remove-unused-alternates` that helps for QCing the inputs before they get to `vcfeval`, so the step probably can't be fully skipped if we wanted to intersect the intervals instead. If you think it's worth better handling the edge cases though, I can try to rework some of the code in that step.